libcontainer/cgroups/fs/cpuacct_test.go

```diff
@@ -95,3 +95,17 @@ func TestCpuacctStatsWithoutUsageAll(t *testing.T) {
 			expectedStats, actualStats.CpuStats.CpuUsage)
 	}
 }
+
+func BenchmarkGetCpuUsageBreakdown(b *testing.B) {
+	path := tempDir(b, "cpuacct")
+	writeFileContents(b, path, map[string]string{
+		"cpuacct.stat": cpuAcctStatContents,
+	})
+
+	for i := 0; i < b.N; i++ {
```

Comment:
nit: please add `b.ResetTimer()` before the `for` loop. This way we won't be taking benchmark preparation time into account.

Reply:
I pushed a commit with this change and did a `git rebase` with the `main` branch.